source/includes/indexes/single-field.rb

```diff
@@ -0,0 +1,37 @@
+require 'mongo'
+
+# Replace the placeholders with your credentials
+uri = "<connection string>"
+
+# Sets the server_api field of the options object to Stable API version 1
+options = { server_api: { version: "1" }}
+
+# Creates a new client and connect to the server
+client = Mongo::Client.new(uri, options)
+
+# start-sample-data
+database = client.use('sample_mflix')
+collection = database[:movies]
+# end-sample-data
+
+# start-index-single
+# Creates an index on the "title" field
+collection.indexes.create_one({ title: 1 })
+# end-index-single
+
+# start-index-single-query
+# Finds a document with the title "Sweethearts"
+filter = { title: 'Sweethearts' }
+doc = collection.find(filter).first
+
+if doc
+  puts doc.to_json
+else
+  puts "No document found"
+end
+# end-index-single-query
```

Comment:
More just a general comment - this has nothing to do with "proving" the index was used ... so why include it?

Reply:
Hi @alexbevi, this section shows users what kind of query the index will impact. It is not supposed to prove the index's creation. Let me know if this doesn't make sense to you.

Reply:
That makes sense, but it's not clear based on the comment that that's the intention (though I guess it's implied).

Maybe a comment of
```ruby
# Finds a document with the title "Sweethearts", which should use the newly created index
```

Reply:
@alexbevi Sounds good, I can add the additional context!